mixer/pkg/perf/setup.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package perf is a helper library for writing Mixer perf tests. It is designed to have a serializable, declarative
+// configuration, that can be run in various execution modes transparently. The package is designed to work
+// seamlessly with the Go perf testing infrastructure.
+//
+// The entry point to the tests is the "Run*" methods. The benchmarks are expected to call one of the methods, passing in
+// testing.B, the test declaration (i.e. Setup struct), an environment variable that contains the ambient template,
```

Comment:
Might be worthwhile to express "environment variable" differently, since it's not OS env vars, right?